assets/blocks/course-progress/edit.js

```diff
@@ -102,17 +109,25 @@ export const EditCourseProgressBlock = ( {
 	);
 };
 
-export default withColorSettings( {
-	barColor: {
-		style: 'background-color',
-		label: __( 'Progress bar color', 'sensei-lms' ),
-	},
-	barBackgroundColor: {
-		style: 'background-color',
-		label: __( 'Progress bar background color', 'sensei-lms' ),
-	},
-	textColor: {
-		style: 'color',
-		label: __( 'Text color', 'sensei-lms' ),
-	},
-} )( EditCourseProgressBlock );
+export default compose(
+	withColorSettings( {
+		barColor: {
+			style: 'background-color',
+			label: __( 'Progress bar color', 'sensei-lms' ),
+		},
+		barBackgroundColor: {
+			style: 'background-color',
+			label: __( 'Progress bar background color', 'sensei-lms' ),
+		},
+		textColor: {
+			style: 'color',
+			label: __( 'Text color', 'sensei-lms' ),
+		},
+	} ),
+	withDefaultColor( {
```

Comment:
I thought that could make sense join it with `withColorSettings`, but I decided to keep them separate because the purposes are different and it could cause maintenance issues in the future.